packages/quantic/force-app/main/default/lwc/quanticResultLabel/quanticResultLabel.js

```diff
@@ -85,6 +88,9 @@ export default class QuanticResultLabel extends LightningElement {
     if (this.label) {
       return this.label;
     }
+    if (this.isKnowledgeArticle) {
+      return KNOWLEDGE;
```

Comment:
This label should be localized, no?

Reply:
I thought so too but given Knowledge a term from salesforce in this case I think leaving it as is makes sense. If I change my org settings to French, "Knowledge" is still "Knowledge" so I did the same here.